### PR TITLE
Create custom user model

### PR DESCRIPTION
**Pull request by brittanylau**
[Task link](https://www.notion.so/uwblueprintexecs/9cb454262ed84c6299c0682c23f92ec5?v=1076bd30711a4a01838becee05f8b5a3&p=6b18f53f88d84ed8892babda3bd136c1)

In this PR:
* created a new app `Accounts`
* created a custom `User` model for Django's auth, which inherits from `AbstractUser`
  * removed the username field and replaced with email for identification

### CODE REVIEW

File: accounts/admin.py [hunk 1]
```diff
@@ -5,7 +5,53 @@
...
+    add_fieldsets = (
+        (
+            None,
+            {
+                "fields": (
+                    "first_name",
+                    "last_name",
+                    "email",
+                    "password1",
+                    "password2",
```
Comment (danielchen115) [inline accounts/admin.py:48]: Is there anywhere we're checking `password1 == password2` or is this done under the hood 😮?

Comment (brittanylau) [reply accounts/admin.py:48]: This is done under the hood! You get this error if they don't match:

Screen Shot 2021-02-06 at 7 04 39 PM

Comment (danielchen115) [reply accounts/admin.py:48]: django is too strong 💪

**Review by danielchen115 — APPROVED** [review]
lgtm! 🚀

**Pull request merged**